R/surv-roc_survival_curve.R

```diff
@@ -140,4 +164,27 @@ roc_curve_survival_impl <- function(truth,
 }
 
 # Dynamically exported
-autoplot.roc_survival_df <- autoplot.roc_df
+autoplot.roc_survival_df <- function(object, ...) {
+
+  `%+%` <- ggplot2::`%+%`
+
+  # Base chart
+  roc_chart <- ggplot2::ggplot(data = object)
+
+  # create aesthetic
+  roc_aes <- ggplot2::aes(
+    x = 1 - specificity,
+    y = sensitivity,
+    color = .eval_time
+  )
+
+  # build the graph
+  roc_chart <- roc_chart %+%
+    ggplot2::geom_path(mapping = roc_aes) %+%
+    ggplot2::geom_abline(lty = 3) %+%
+    ggplot2::coord_equal() %+%
+    ggplot2::theme_bw() %+%
+    ggplot2::scale_color_viridis_c()
```

Comment:
Let the user pick the color palette and theme

Reply:
If this code is also in the ROC function, we should get rid of it. I've been frustrated by this in the past.

Reply:
done in [`3cc507c` (#407)](https://github.com/tidymodels/yardstick/pull/407/commits/3cc507cf0864667591934b7d5c3c3f37713df9b2)